lsp-mode.el

```diff
@@ -4970,7 +4982,8 @@ Also, additional data to attached to each candidate can be passed via PLIST."
                                                           bounds-start)))
                                            it))))
                         (markers (list bounds-start (copy-marker (point) t)))
-                        (prefix (buffer-substring-no-properties bounds-start (point))))
+                        (prefix (buffer-substring-no-properties bounds-start (point)))
+                        (lsp--capf-no-reordering t))
```

Comment:
based on @payneca comment I rather rever that and have it yes/no instead of have it only initial sort.

Reply:
I still would like to keep this for initial sort only.
The argument is because of this is initial sort right after we receive results from server, so server should probably have better ordering than client. We still need filtering though.

Here is our full options https://github.com/emacs-lsp/lsp-mode/issues/1883#issuecomment-655186783

In this special case, I think we should change from server side, so it can work with more clients, instead of just relying on our client implementation behavior.